docs/wiki/guides/current/Magnetometer.md

```diff
@@ -92,35 +82,56 @@ When we measure the magnetic field at a particular point on earth, we are measur
+
+## Mounting a Magnetometer
+
+It is a whole lot easier to fix orientation problems if the magnetometer is physically mounted in the same plane as the FC (e.g., flat to the frame).
+
+The sensor in most standalone Mag modules, e.g. the GY-271 board, is soldered on the top of the board, usually with the Z axis pointing upwards and X forwards (in the direction of the arrow). These kinds of boards usually _do not_ require a 'flip' orientation. They are best mounted:
+
+- centrally
+- in the same plane of the FC, typically both devices 'flat to the frame'
+- as far as possible, away from motors, other metallic onjects, and high-current wires
+- with the X axis pointing forward, directly to the nose of the quad, Y left and Z up
+
+The sensor in most GPS modules is mounted _upside-down_. The Z axis points _downwards_. They require a _'flip'_ orientation.
+
+Many GPS modules are mounted with some pitch forward angle. This will put a forward pitch on the Magnetometer, and that means our 'stock' orientation corrections will not work, and that you will need to set up custom Mag orientation in the CLI. It is a whole lot easier, if the Mag ins on the GPS, when the GPS is mounted flat to the FC.
+
+Note that the sensor can be soldered to the board (GPS or Standalone) with it's X axis facing forward, or backwards, or left, or right, and you may have mounted the module facing forward, sideways or backwards on the quad. Hence the X axis can often point in the wrong direction with respect to the nose of the quad. This is why you should always check the X axis orientation before using the Mag.
+
+The `Mag alignment` setting in Configurator (`align_mag` in the [CLI](/docs/wiki/configurator/cli-tab) should be used to correct the orientation. Standard corrections (e.g. `CW90` to rotate the axes 90° clockwise) are provided for common alignment problems. If the module is tilted backwards, or at an unusual angle, a `custom` orientation correction will be required. See the `Magnetometer Orientation` section, below, for more information.
 
 ## Hardware and Connection
 
-Betaflight's build system must include `Magnetometers`, from the dropdown in the cloud build, or `-DUSE_MAG` for local builds, otherwise there will be no Mag support in the firmware. There is no need to explicitly request the inclusion of any specific driver for a specific mag, because all supported Mag drivers will be included in the build.
+Betaflight's build system must include `Magnetometers`, from the dropdown in the cloud build, or `-DUSE_MAG` for local builds, otherwise there will be no Mag support in the firmware. The Mag build option includes drivers for all supported Mgnetometers.
```

Comment:
```suggestion
Betaflight's build system must include `Magnetometers`, from the dropdown in the cloud build, or `-DUSE_MAG` for local builds, otherwise there will be no Mag support in the firmware. The Mag build option includes drivers for all supported Magnetometers.
```

Reply:
thanks